packages/ui/src/ui/pages/components/tabs/nodes/tables.tsx

```diff
@@ -468,9 +513,24 @@ const nodesTableProps = {
                             column="memory"
                             title="Memory"
                             options={[
-                                {column: 'memory', title: 'Progress', withUndefined: true},
-                                {column: 'memory_usage', title: 'Usage', withUndefined: true},
-                                {column: 'memory_limit', title: 'Limit', withUndefined: true},
+                                {
+                                    column: 'memory',
+                                    title: 'Progress',
+                                    withUndefined: true,
+                                    allowUnordered: true,
+                                },
+                                {
+                                    column: 'memory_usage',
+                                    title: 'Usage',
+                                    withUndefined: true,
+                                    allowUnordered: true,
+                                },
+                                {
+                                    column: 'memory_limit',
+                                    title: 'Limit',
+                                    withUndefined: true,
+                                    allowUnordered: true,
```

Comment:
I don't really like that we have a lot of blocks with duplicate parameters `withUndefined: true, allowUnordered: true`

I suggest putting them in a constant and adding them via `{title: '...', ...MY_CONST}`